### ISSUE TITLE
When deploying i get an Error >> TypeError: Cannot read property 'includes' of null

### ISSUE THREAD

**Issue by thvd** [vlocityinc/vlocity_build#156]
When I run `packDeploy` i get this error sometimes: 
```js

  Error >> TypeError: Cannot read property 'includes' of null
    at module.exports.DataPacksErrorHandling.getMatchingError (....../node_modules/vlocity/lib/datapackserrorhandling.js:68:34)
    at module.exports.DataPacksErrorHandling.getSanitizedErrorMessage (....../node_modules/vlocity/lib/datapackserrorhandling.js:24:49)
    at module.exports.DataPacksJob.deployPack (....../node_modules/vlocity/lib/datapacksjob.js:1796:71)
    at process._tickCallback (internal/process/next_tick.js:68:7)  


```

And on the end i see this error: 

```js
Deploy error:
SObject_CustomFieldMap/04bdb4-859ec5-cc77d3-8c9831-606662d6f -- DataPack >> 04bdb4-859ec5-cc77d3-8c9831-606662d6f -- Error Message -- No error message from server

```

And if i search for `SObject_CustomFieldMap/04bdb4-859ec5-cc77d3-8c9831-606662d6f` i find this one: 

```js

  Setting to Deploy Separate >> SObject_CustomFieldMap/04bdb4-859ec5-cc77d3-8c9831-606662d6f --- Unhandled Error Occurred: Delete failed. First exception on row 0 with id aCM0D0000008hD1WAI; first error: UNABLE_TO_LOCK_ROW, unable to obtain exclusive access to this record or 1 records: aCM0D0000008hD1WAI: []  


```

All using vlocity version `v1.8.5 vlocity_cmt 900.163`

**Comment by rutlabaga** [comment]
This should be fixed in v1.8.6.

**Comment by thvd** [comment]
Thanks for the fast fix. Will test tomorrow.
